src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+/*
+ * Represents a person's block number in the address book
+ */
+
+public class Block {
+	String block = new String();
```

Comment:
Who needs access to `block`? Which access modifier would be more suitable for `block` here?